tests/e2e/basic-sauce-config.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+metadata:
+  name: Feature XYZ
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - tests/fixtures/basic/
+suites:
+  - name: "saucy test"
```

Comment:
Would be great if you could use unique names when using multiple suites. With the upcoming concurrency support, having unique names will most likely be mandatory.